kbase-extension/static/kbase/js/api/upa.js

```diff
@@ -108,13 +108,65 @@ define([
             return deserial;
         };
 
+        var changeUpaVersion = function(upa, newVersion) {
+            if (!isUpa(upa)) {
+                throw {
+                    error: upa + ' is not a valid upa, so its version cannot be changed!'
+                };
+            }
+            var newUpa = upa.replace(/^(.+\/)(\d+)$/, '$1' + newVersion);
+            return newUpa;
+        };
+
+        var serializeAll = function(upas) {
+            if (typeof upas === 'string') {
+                return serialize(upas);
+            }
+            else if (Array.isArray(upas)) {
+                var serialArr = [];
+                upas.forEach(function(upa) {
+                    serialArr.push(serialize(upa));
+                });
+                return serialArr;
+            }
```

Comment:
I never remember what's available...do we have map in our environment? If so, this could just be:

return upas.map( function(upa) { return serialize(upa) } );

Not actually necessary to fix, just a little prettier. Even better when we could have

return upas.map( upa => serialize(upa) )

Reply:
Yeah, map's available. Not sure why I never think of it. I'll switch that out.